### ISSUE TITLE
In the final solution notebook the part where cross_val_score is being used is not working the error it shows is target is not defined.I have been looking for same but didn't found anything solid.

### ISSUE THREAD

**Issue by piyushmanglani08** [minsuk-heo/kaggle-titanic#1]

**Comment by MichaelThomasJohnson** [comment]
I have found the same thing

**Comment by piyushmanglani08** [comment]
Yeah but i found its solution that you have to re run each and every cell
line by line to make it work try this if it helps.

**Comment by MichaelThomasJohnson** [comment]
I also tried that and for some reason mine didnt work. What i did is right before MODELING i ran this.
train_data.fillna(0, inplace=True). Fills the 3 blank cells in Embarked and Title. Might just run back through it again so i dont mess with the data but this was a quick solution to move on. Like i said that is not the proper way to go about a real world scenario

**Comment by piyushmanglani08** [comment]
Yes that is true.but to just see it working it helps.
